index.html

```diff
@@ -167,86 +173,131 @@ <h2>Verifiable Credential</h2>
 
     <section>
       <h2>Securing JSON-LD</h2>
+      <p>
+        The use of Verifiable Credentials often involves the representation and exchange of structured 
+        data in the form of JSON-LD. While JSON-LD provides a flexible and extensible format for 
+        describing data, it is important to note that it also provides a linkage between the data structure
+        and semantic meaning of data.
+        This section outlines how JSON-LD encoded claimset can be secured using either JOSE or COSE.
```

Comment:
```suggestion
        This section outlines how JSON-LD encoded payload can be secured using either JOSE or COSE.
```

Reply:
> The JWT Claims Set represents a JSON object whose members are the claims conveyed by the JWT. 
https://www.rfc-editor.org/rfc/rfc7519#section-4

Reply:
@OR13,

Calling the payload a JWT Claims Set means that when your payload has this:

```js
{
  "@context": ["<credentials v2 context>", "something else"],
  "iss": {"this": "is different from the IANA registry because it has nothing to do with it"},
  "sub": {"this": "is", "not": "the registered 'sub' claim"},
  "...so on": "foo"
}
```

Then the expectation is that processors should use the [IANA JWT Claims registry](https://www.iana.org/assignments/jwt/jwt.xhtml) to understand `iss`, `sub`, and so on. This is wrong and will cause confusion.